core/app/models/spree/user_return_to_storer.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Spree
+  # This service object is responsible for storing the current path into
+  # into `session[:spree_user_return_to]` for redirects after successful
+  # user/admin authentication.
+  class UserReturnToStorer
```

Comment:
This name is a bit hard to understand maybe, at least for me at the first look. I immediately thought about Returns and I'm wondering if we can use another terminology, which conflicts less with other elements of the Solidus domain, like `UserLastLocationStorer` maybe? I think it could be better even because this proposed name is not connected with how we'll use it, while `ReturnTo` is. Not a strong opinion here, what's yours?

Reply:
My strong opinion here is that I don't like that name either 😄 

I think `UserLastLocationStorer` is good and better than mine, but it's a bit long. What about `UserLastUrlStorer` or `UserLocationStorer`?

As soon as we agree on a name I'll revisit the PR with the other suggestions as well... so much namespace and filenames to change 😄

Reply:
I think `UserLastUrlStorer` is fine. Let's see if we have some other opinion here :)